discovery-infra/test_infra/controllers/node_controllers/node.py

```diff
@@ -102,6 +102,10 @@ def restart(self):
         self.shutdown()
         self.start()
 
+    def restart_service(self, service):
+        logging.info("Restarting service: %s on host %s", service, self.name)
+        self.run_command(f'sudo systemctl restart {service}.service || true')
```

Comment:
I know that is already being used by `is_service_active` and `kill_service` but  I don't like the `|| true`. In case there was a fatal error I'd like to know about it. In case a calling function would like to ignore it, then we can wrap it with a corresponding `supress`.
For example, I'd like to know if the service name I provided isn't available at all.

Reply:
Fixed it. Thanks!